content/en/blog/2023/sig-network-blixt-ebpf-rust-in-kubernetes.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: Blixt - A load-balancer written in Rust, using eBPF, born from Gateway API
+date: 2023-11-15
+slug: blixt-load-balancer-rust-ebpf-gateway-api
+author: "Shane Utt (Kong), Andrew Stoycos (Red Hat)"
+---
+
+In [SIG Network][signet] we now have a layer 4 load-balancer named [Blixt][blixt]. This
+project started as a fun experiment using emerging technologies and is intended
+to become a utility for CI and testing to help facilitate the continued
+development of [Gateway API][gwapi]. Are you interested in developing networking
+tools in [Rust][rust] and [eBPF][ebpf]? Or perhaps you're specifically
+interested in Gateway API? We'll tell you a bit about the project and how it
+might benefit you.
+
+[signet]:https://github.com/kubernetes/community/tree/master/sig-network
+[blixt]:https://github.com/kubernetes-sigs/blixt
+[gwapi]:https://github.com/kubernetes-sigs/gateway-api
```

Comment:
Potentially, link to the page about the gateway API inside the Kubernetes documentation (once it exists).